src/cmap/connect.ts

```diff
@@ -377,7 +371,7 @@ function makeConnection(options: MakeConnectionOptions, _callback: Callback<Stre
     socket = net.createConnection(parseConnectOptions(options));
   }
 
-  socket.setKeepAlive(keepAlive, keepAliveInitialDelay);
+  socket.setKeepAlive(true, 300000);
```

Comment:
LGTM, Just calling out a reminder the spec mentions linking documentation, since we cannot set `tcp_keepalive_cnt` is that something that can be configured at the system level?

Reply:
I've updated the docs changes needed to note that we should document what we set them to now, and if they need to be configured at the system level we should link to https://www.mongodb.com/docs/manual/faq/diagnostics/#does-tcp-keepalive-time-affect-mongodb-deployments